src/main/resources/db/changelog/202401081559-add-claimant-owner-column.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet id="202401081559-1" author="ojanus">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="cp_owner" columnName="claimant_owner"/>
+            </not>
+        </preConditions>
+
+        <addColumn tableName="cp_owner">
+            <column name="claimant_owner" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <addForeignKeyConstraint baseTableName="cp_owner"
+            baseColumnNames="claimant_owner"
+            referencedTableName="cp_owner"
+            referencedColumnNames="account"
+            constraintName="fk_cp_owner_claimant_owner"
+            onDelete="CASCADE" />
```

Comment:
Are the columns in this FK constraint backwards? If we delete the anonymous owner that has been claimed and has the `claimant_owner` value populated, won't this delete the claimant owner as well? That would be problematic because the claimant owner is the owner the consumers should be registered to. If the claimant owner gets deleted, we should cascade and delete the anonymous owners that were claimed by that owner.

@nikosmoum just want a double check that I'm right on this

Reply:
This is correct. When referenced row gets deleted, base will get deleted as well, but I double checked to be sure.

Reply:
Yes, the current code is correct. If you delete the destination owner, both get deleted. If you delete the anonymous owner, the destination owner isn't getting deleted

Reply:
Thanks for double checking that guys!